test/tests/component.test.js

```diff
@@ -57,6 +61,13 @@ describe('Custom components rendering', () => {
       done()
     })
   })
+
+  it('Custom progressBarColor is used', (done) => {
+    request.get(uri, (err, res, body) => {
+      expect(body).to.contain('background-color:#000;')
```

Comment:
Everying looks great, minor minor things, is there a possibility than #000 could be included in the CSS elsewhere? Perhaps #c0ffee or something?